src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -324,7 +324,6 @@ def create(self, no_registry=False):
             resource_group_name=self.resource_group.name,
             image=self.image,
             managed_env=self.env.get_rid(),
-            target_port=self.target_port,
```

Comment:
`create_containerappsjob` has no argument `target_port`, so remove this input, otherwise will cause some pylint warning.